Plugins/Bitbucket/BitbucketPullRequestForm.cs

```diff
@@ -29,6 +29,12 @@ public BitbucketPullRequestForm(BitbucketPlugin plugin, ISettingsSource settings
             InitializeComponent();
             Translate();
 
+            if (gitUiCommands == null)
```

Comment:
❓ Does this only affect the Bitbucket plugin? How do other forms handle the same situation? Can we not update the translation app to provide this value?

Reply:
No other forms have caused NRE.
A lot of forms have a default ctor (often marked as `private`), a lot of forms check for `gitUiCommands` instance being `null`.
We certainly need to align all API to be consistent. 

It just occurred to me, that requiring all forms to provide a default ctor that calls `Translate()` may address the issue I described above.
Thoughts?

Reply:
> It just occurred to me, that requiring all forms to provide a default ctor that calls `Translate()` may address the issue I described above.

It seems like alternatives are now available that would remove the need for TranslateApp altogether. I haven't seen something like this needed in the past; why not just use the compile-time XLF tools?

Reply:
That's something to research too 😀 
Right now it is to fix what's broken, so we can move on.